R/pkg/R/mllib.R

```diff
@@ -124,24 +138,21 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
 #' summary(model)
 #' }
 #' @note glm since 1.5.0
+#' @seealso \link{spark.glm}
 setMethod("glm", signature(formula = "formula", family = "ANY", data = "SparkDataFrame"),
           function(formula, family = gaussian, data, epsilon = 1e-6, maxit = 25) {
             spark.glm(data, formula, family, tol = epsilon, maxIter = maxit)
           })
 
-#' Get the summary of a generalized linear model
-#'
-#' Returns the summary of a model produced by glm() or spark.glm(), similarly to R's summary().
+#  Returns the summary of a model produced by glm() or spark.glm(), similarly to R's summary().
```

Comment:
I get this is intentional, but I'd suggest adding extra empty newline between `#` & `#'` line - it's very easy to newcomers to make mistakes copy/paste doc